### PR TITLE
Stop making :upcoming-fresh tags; use osg-upcoming-minefield for the :fresh tags

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: .github/workflows/release-image-builds.yml [hunk 1]
```diff
@@ -96,13 +88,13 @@ jobs:
...
-      - run: docker pull docker.pkg.github.com/$GITHUB_REPOSITORY/${{ matrix.image }}:${{ matrix.yum_repo_prefix }}minefield-${{ needs.get-timestamp.outputs.dtag }}
+      - run: docker pull docker.pkg.github.com/$GITHUB_REPOSITORY/${{ matrix.image }}:upcoming-minefield-${{ needs.get-timestamp.outputs.dtag }}
       - name: Build ${{ matrix.image}} image
         uses: docker/build-push-action@v1
         with:
           repository: opensciencegrid/${{ matrix.image }}
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          tags: ${{ matrix.yum_repo_prefix }}fresh, ${{ matrix.yum_repo_prefix }}${{ needs.get-timestamp.outputs.dtag }}
+          tags: fresh, upcoming-${{ needs.get-timestamp.outputs.dtag }}
```
Comment (brianhlin) [inline .github/workflows/release-image-builds.yml:100]:
```suggestion
          tags: fresh, ${{ needs.get-timestamp.outputs.dtag }}
```

I think we should only ship `fresh` and timestamp tags

**Review by brianhlin — COMMENTED** [review]
Looks mostly good but I don't think we should ship new `upcoming-<TIMESTAMP>` tags

File: .github/workflows/release-image-builds.yml (hunk 1 above)
Comment (matyasselmeci) [reply .github/workflows/release-image-builds.yml:100]: Oops, didn't see that.

**Review by brianhlin — APPROVED** [review]
LGTM but I'd like to hold off on merging to avoid using up our Travis credits. I'll rebase #56 on this and we can merge them both together

**Pull request merged**